test/e2e/split/main.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"math"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+var topDescribeRE = regexp.MustCompile(`var _ = Describe\("(.+)", func\(.*`)
+
+func main() {
+	var numChunks, printChunk int
+	flag.IntVar(&numChunks, "chunks", 1, "Number of chunks to create focus regexps for")
+	flag.IntVar(&printChunk, "print-chunk", 0, "Chunk to print a regexp for")
+	flag.Parse()
+
+	if printChunk >= numChunks {
+		log.Fatalf("the chunk to print (%d) must be a smaller number than the number of chunks (%d)", printChunk, numChunks)
+	}
+
+	dir := flag.Arg(0)
```

Comment:
nit: we may need validation to ensure this directory has been supplied as a flag?